doc/source/data/examples/nyc_taxi_basic_processing.ipynb

```diff
@@ -348,24 +390,32 @@
     "For the NYC taxi dataset, instead of reading individual per-month Parquet files, we can read the entire 2009 directory.\n",
     "\n",
     "```{warning}\n",
-    "This will be a lot of data (~5.6 GB on disk, ~14 GB in memory), so be careful trigger full reads on a limited-memory machine! This is one place where Datasets' semi-lazy reading comes in handy: Datasets will only read one file eagerly, which allows us to inspect a subset of the data without having to read the entire dataset.\n",
+    "This could be a lot of data (downsampled with 0.01 ratio leads to ~50.2 MB on disk, ~147 MB in memory), so be careful trigger full reads on a limited-memory machine! This is one place where Datasets' semi-lazy reading comes in handy: Datasets will only read one file eagerly, which allows us to inspect a subset of the data without having to read the entire dataset.\n",
     "```"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 10,
-   "id": "9cc641b3",
+   "id": "4a1fa8ab",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "(_get_read_tasks pid=9272) 2022-07-25 14:29:30,732\tINFO parquet_datasource.py:323 -- Parquet input size estimation took 28.51 seconds.\n"
```

Comment:
@ericl @c21 28.51 seconds for size estimation on a small dataset (50 MB on disk, 150 MB in memory) seems really high?

Reply:
Is the notebook expected to run on user's local machine? I feel this might due to the network speed to read S3 files into local machine.

I tried locally on my laptop, and I found the spent time varied no matter how large the file size is.

Downsampled data used here (sampling took 6.54 seconds):

```
>>> ds = ray.data.read_parquet([
...     "s3://air-example-data-2/ursa-labs-taxi-data/downsampled_2009_01_data.parquet",
...     "s3://air-example-data-2/ursa-labs-taxi-data/downsampled_2009_02_data.parquet"])
⚠️  The number of blocks in this dataset (2) limits its parallelism to 2 concurrent tasks. This is much less than the number of available CPU slots in the cluster. Use `.repartition(n)` to increase the number of dataset blocks.
>>> (_get_read_tasks pid=70675) 2022-07-25 15:33:27,142	INFO parquet_datasource.py:323 -- Parquet input size estimation took 6.54 seconds.
```

Full data (sampling took within 5 seconds, so no printing):

```
>>> ds = ray.data.read_parquet([
...     "s3://ursa-labs-taxi-data/2009/01/data.parquet",
...     "s3://ursa-labs-taxi-data/2009/02/data.parquet"])
⚠️  The blocks of this dataset are estimated to be 2.0x larger than the target block size of 512 MiB. This may lead to out-of-memory errors during processing. Consider reducing the size of input files or using `.repartition(n)` to increase the number of dataset blocks.
>>> 
```

Reply:
@jiaodong - sorry for the trouble, could you help remove this log? We disable Parquet sampling by default in 2.0 for the sake of stability (https://github.com/ray-project/ray/pull/27034). So this log is not printed out now.

Reply:
sure i can remove it